vendors/watteco/drivers/hygrotemp'o/driver.yaml

```diff
@@ -0,0 +1,17 @@
+#Formal driver name
+name: hygrotemp'o driver
+# Simple description of the driver functionalities
+description: Decoder for hygrotemp'o
+# Mandatory - id of the company that developed the driver
+# This id will be used to construct the id of the driver (Should be in lowercase, no special characters, maximum 8 characters)
+producerId: watteco
+# Email of the developer
+developerEmail: lena.mabille@actility.com
+# In case of a driver per model: `<vendorId>:<modelName>:<modelVersion>`
+# In case of a driver for multiples model: `<vendorId>:generic:<genericVersion>`
+# Needed for linking the model with a specific driver -> must be the same one used in model.yaml in the corresponding model
+# You should have only one protocolId per driver
+protocolId: wwatteco:th:4
```

Comment:
```suggestion
protocolId: watteco:hygrotempo:4
```

Reply:
there is already a watteco TH driver and model. what about using the device name in the protocolId to refer to the driver? 
If you are agree with me, than the model.yaml should be also updated here: https://github.com/actility/device-catalog/blob/main/vendors/watteco/models/hygrotemp'o_v4/model.yaml#L36